src/Components/Assets/AssetType/HL7Monitor.tsx

```diff
@@ -19,6 +19,8 @@ const HL7Monitor = (props: HL7MonitorProps) => {
   const { assetId, asset } = props;
   const [assetType, setAssetType] = React.useState("");
   const [middlewareHostname, setMiddlewareHostname] = React.useState("");
+  const [middlewareHostname_error, setMiddlewareHostname_error] =
```

Comment:
@patelaryan7751  snake_case with camelCase naming convention?

Reply:
i wrote the error at end and field name separated by underscore just to add similarity.

Reply:
@patelaryan7751 Refactor code to use `useReducer` hook for state management